scripts/__input_initialize/__input_initialize.gml

```diff
@@ -170,13 +170,36 @@ function __input_initialize()
         ++_p;
     }
     
+    //DualSense haptic trigger effect types
+    enum INPUT_TRIGGER_EFFECT
+    {
+        TYPE_OFF,
+        TYPE_FEEDBACK,
+        TYPE_WEAPON,
+        TYPE_VIBRATION,
+    }
+    
+    //DualSense haptic trigger effect states
+    enum INPUT_TRIGGER_STATE
```

Comment:
Is this enum expected to be public? If not then please prefix with `__` (same with the enum members as well i.e. `__INPUT_TRIGGER_STATE.__EFFECT_OFF`)

Reply:
public, returned by trigger state getter